tools/e2etesting/steps/runtests.yml

```diff
@@ -131,6 +131,7 @@ steps:
     NESTED_EDGE_FLAG: "$(NestedEdgeFlag)"
     NESTED_EDGE_SSH_CONNECTIONS: "$(NestedEdgeSshConnection)"
     SP_TENANT_ID: '$(SP-TENANT-ID)'
+    SUBSCRIPTION_ID: '$(SUBSCRIPTION-ID)'
```

Comment:
We may have `PCS-` environment variables for subscription and tenant and those should be used instead of new `SP-TENANT-ID` and `SUBSCRIPTION-ID` that are consumed by the tests.

Reply:
Renamed `SUBSCRIPTION_ID` to `PCS_SUBSCRIPTION_ID`.
Renamed `SP-TENANT-ID` to `PCS-TENANT-ID`.
Renamed `RESOURCE_GROUP_NAME` to `PCS_RESOURCE_GROUP`.